admin/services/pupil-census-processing.service.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const schoolDataService = require('../services/data-access/school.data.service')
+const pupilCensusImportDataService = require('../services/data-access/pupil-census-import.data.service')
+const pupilCensusProcessingService = {}
+
+/**
+ * Processes and bulk imports pupil census
+ * @param csvData
+ * @return {Promise<void>}
+ */
+pupilCensusProcessingService.process = async(csvData) => {
+  // Fetch all school for pupil records
+  let schoolDfeNumbers = csvData.map(r => `${r[0]}${r[1]}`)
+  // filter duplicate entries
+  schoolDfeNumbers = schoolDfeNumbers.filter((item, pos, self) => {
```

Comment:
Do you think it would be easier and more obvious to filter by using a `Set`?  Or `R.uniq()`?

Reply:
@jon-shipley I think R.uniq would be the cleanest way. Updated